.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: Release
+
+
+on:
+  push:
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build project
+        run: zip templates.zip `find * -iname "*.txt"`
```

Comment:
"submission-template" or something that has more meaning when it's unzipped on disk?

Reply:
true or even `IDR_submission_templates.zip`?

Reply:
https://github.com/sbesson/idr0000-lastname-example/releases/tag/v1.0.1